back-end/photo_app/admin.py

```diff
@@ -1,3 +1,5 @@
 from django.contrib import admin
+from .models import Photo
+
+admin.site.register(Photo)
```

Comment:
I still don't see the Photos in the admin panel, make sure you aren't missing any other steps ;)